packages/fluent/scss/input/_variables.scss

```diff
@@ -0,0 +1,617 @@
+@use "../core/color-system/" as *;
+@use "../core/_variables.scss" as *;
+@use "../icon/_variables.scss"as *;
+@use "../button/_variables.scss"as *;
+
+/// Width of the input.
+/// @group input
+$kendo-input-width: 100% !default;
+
+/// Width of the border around the input.
+/// @group input
+$kendo-input-border-width: 1px !default;
+
+// Width of the border around the input when focused.
+$kendo-input-focus-border-width: 2px !default;
+
+/// Height of he border around the input.
+/// @group input
+$kendo-input-border-height: calc( $kendo-input-border-width * 2 ) !default;
+
+/// Border radius of the input.
+/// @group input
+$kendo-input-border-radius: 2px !default;
```

Comment:
I think here we should rather use var( --kendo-border-radius-md, 0 ) , so that if the global border-radius is changed, the input's border radius will be changed as well.